packages/starlight/components/Search.astro

```diff
@@ -95,11 +95,16 @@ const pagefindTranslations = {
       closeBtn.addEventListener('click', closeModal);
 
       // Listen for `/` and `cmd + k` keyboard shortcuts.
+      const inputs = ["input", "select", "textarea"];
+      const isInput =
+        document.activeElement &&
+        inputs.indexOf(document.activeElement.tagName.toLowerCase()) !== -1;
```

Comment:
Thanks for flagging this and opening the PR — appreciate it!

Would you mind sharing how you tested this? In the deploy preview, if I focus one of the `<select>` menus in the site header and press `/`, the search modal is still triggered.

Reply:
Oh, I see the issue now! The `isInput` check runs once when the script initialises and never again, whereas we want to run it on each `keydown` event. Will suggest some fixes.

Reply:
Those make sense to me. I've commited them. Let me know if you want me to squash and rebase this PR?